docs/docs/videos.md

```diff
@@ -168,6 +168,6 @@ Walk-through by [Andres Suarez](https://github.com/zertosh) on how [SoundCloud](
 
 ### Introducing React Native (+Playlist) - React.js Conf 2015
 
-<iframe width="650" height="366" src="https://www.youtube.com/embed/KVZ-P-ZI6W4?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr" frameborder="0" allowfullscreen></iframe>
+<iframe width="650" height="366" src="https://www.youtube-nocookie.com/embed/KVZ-P-ZI6W4?list=PLb0IAmt7-GS1cbw4qonlQztYV1TAW0sCr" frameborder="0" allowfullscreen></iframe>
```

Comment:
For future note, on a YouTube video, click on "Share" -> "Embed" -> "Show More" -> "Enable privacy-enhanced mode", you'll get this same youtube-nocookie.com link. [This YouTube support page](https://support.google.com/youtube/answer/171780?expand=PrivacyEnhancedMode#privacy) provides more information. Backstory is that it was added at the request of the White House so they could embed YouTube videos on whitehouse.gov without a cookie being immediately set by YouTube for loading the interstitial (before the video is actually loaded and played).